lib/primer/view_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Provides a convenient shorthand for components.
+module Primer::ViewHelper
+  Components = Hash.new do |hash, component|
+    hash[component] = "Primer::#{component.to_s.camelize}Component".constantize
```

Comment:
for now I guess this should be fine, but I imagine we'll have some more complex namespaces in the future